packages/ui/src/memberships/components/MemberListFilters/MemberListFilters.tsx

```diff
@@ -0,0 +1,155 @@
+
+const filterReducer = (filters: MemberListFilter, action: Action): MemberListFilter => {
+  switch (action.type) {
+    case 'clear':
+      return MemberListEmptyFilter
+
+    case 'change':
+      return { ...filters, [action.field]: action.value }
+  }
+  return filters
+}
+
+const CouncilOpts = {
+  All: null,
+  Yes: true,
+  no: false,
+}
+
+export const MemberListEmptyFilter: MemberListFilter = {
+  search: '',
+  role: null,
+  concil: CouncilOpts.All,
+  onlyVerified: false,
+  onlyFounder: false,
+}
+
+interface MemberListFiltersProps {
+  roles: { [k: string]: string }
+  onApply?: (value: MemberListFilter) => void
+}
+export const MemberListFilters = ({ roles, onApply }: MemberListFiltersProps) => {
+  const [filters, dispatch] = useReducer(filterReducer, MemberListEmptyFilter)
+  const { search, role, concil, onlyVerified, onlyFounder } = filters
+
+  return (
+    <MemberListFiltersContainer>
+      <ClearBtn
+        onClick={() => {
+          dispatch({ type: 'clear' })
+          onApply?.(MemberListEmptyFilter)
+        }}
+      >
+        Clear all Filters
+      </ClearBtn>
+
+      <Fields>
+        <InputComponent icon={<SearchIcon />}>
+          <InputText
+            placeholder="Search"
+            value={search}
+            onChange={(evt: ChangeEvent<HTMLInputElement>) => {
+              const { value } = evt.target
+              dispatch({ type: 'change', field: 'search', value })
+            }}
+          />
+        </InputComponent>
+
+        <SimpleSelect
+          title="Roles"
+          options={{ All: null, ...roles }}
+          value={role}
+          onChange={(value) => {
+            dispatch({ type: 'change', field: 'role', value })
+          }}
+        />
+        <SimpleSelect
+          title="Council Members"
+          options={CouncilOpts}
+          value={concil}
+          onChange={(value) => {
+            dispatch({ type: 'change', field: 'concil', value })
+          }}
+        />
+        <TogglableIcon
+          value={onlyVerified}
+          onChange={(value) => {
+            dispatch({ type: 'change', field: 'onlyVerified', value })
+          }}
+        >
+          <VerifiedMemberIcon />
+        </TogglableIcon>
+        <TogglableIcon
+          value={onlyFounder}
+          onChange={(value) => {
+            dispatch({ type: 'change', field: 'onlyFounder', value })
+          }}
+        >
+          <FounderMemberIcon />
+        </TogglableIcon>
+
+        <ButtonPrimary onClick={onApply && (() => onApply(filters))}>Apply</ButtonPrimary>
+      </Fields>
+    </MemberListFiltersContainer>
+  )
+}
+
+const ClearBtn = styled.button`
+  border: none;
+  margin: 0;
+  padding: 0.5rem;
+  width: auto;
+  overflow: visible;
+  cursor: pointer;
+  background: transparent;
+  color: ${Colors.Grey};
+  font: inherit;
+  line-height: normal;
+  -webkit-font-smoothing: inherit;
+  -moz-osx-font-smoothing: inherit;
+  -webkit-appearance: none;
```

Comment:
This should be `appearance`. AFAIC styled-components uses [stylist](https://stylis.js.org/) for automatic vendor prefixing and it properly prefixes `appeareance` to:

```css
{
	-webkit-appearance: none;
	-moz-appearance: none;
	-ms-appearance: none;
	appearance: none;
}
```